com.unity.cinemachine/Runtime/Core/InputAxisControllerBase.cs

```diff
@@ -238,13 +223,25 @@ static int GetControllerIndex(List<Controller> list, IInputAxisOwner owner, stri
             in IInputAxisOwner.AxisDescriptor axis, Controller controller)
         { 
         }
+        
+        /// <summary>
+        /// Called when the component is required to be reset.
+        /// </summary>
+        protected virtual void OnResetComponent() {}
+        
+        /// <summary>
+        /// Called when the inputs are updated
+        /// </summary>
+        protected virtual void OnUpdateInputs() {}
```

Comment:
I like what you're doing, but I don't like the implementation.  I re-implemented it in my push.  Please have a look at it.